app/components/containers/containersController.js

```diff
@@ -12,6 +12,7 @@ angular.module('containers', [])
     $scope.sortReverse = ($scope.sortType === sortType) ? !$scope.sortReverse : false;
     $scope.sortType = sortType;
   };
+  $scope.URLPublish = EndpointProvider.endpointURLPublish() || '0.0.0.0';
```

Comment:
Yeah, that was the idea, but I can't send a empy value to api for this field
I fall back to init URLPublish with '_' for socket endpoints, and add condition fot htat in containers view
If you have an other solution, i take !